### ISSUE TITLE
ModuleNotFoundError: No module named 'appium'

### ISSUE THREAD

**Issue by gauthierbuttez** [Nuitka/Nuitka#681]
Hi,

I am using python 3.7.7 and Nuitka on windows 10:

```
>>> nuitka --version
0.6.7
Python: 3.7.7 (tags/v3.7.7:d7c567b08f, Mar 10 2020, 10:41:24) [MSC v.1900 64 bit (AMD64)]
Executable: C:\Users\Nino\AppData\Local\Programs\Python\Python37\python.exe
OS: Windows
Arch: x86_64
```

When I run my exe in the dist folder, I get this error message:

```
Traceback (most recent call last):
  File "E:\DropboxBACKUP14112018\Cff\Python\PhoneBot_with_lock\phonebot.dist\phonebot.py", line 7, in <module>
  File "mymodules.py", line 18, in init mymodules
ModuleNotFoundError: No module named 'appium'
```
So I run python shell in order to test 'import appium' and see if it was correctly installed:

```
E:\>python
Python 3.7.7 (tags/v3.7.7:d7c567b08f, Mar 10 2020, 10:41:24) [MSC v.1900 64 bit (AMD64)] on win32
Type "help", "copyright", "credits" or "license" for more information.
>>> import appium
>>>
```

And it did import without any issue. So it appium is correctly installed.

This is the nuitka command I use to produce my exe:
`python -m nuitka --standalone --plugin-enable=numpy=xarray --show-progress --show-scons --follow-imports --recurse-all --windows-icon=E:\DropboxBACKUP14112018\Cff\Python\PhoneBot\favicon_phonebot.ico phonebot.py`

And this is the log of compilation(too long to paste it here directly):

[https://github.com/gauthierbuttez/public/blob/master/nuitka_log.txt](https://github.com/gauthierbuttez/public/blob/master/nuitka_log.txt)

Do you have any idea how can I fix this "no module" issue?

**Comment by kayhayen** [comment]
Can you make a minimal reproducer, maybe as small as `import appium`, so we have evidence that your `init mymodules` is not to blame?

**Comment by gauthierbuttez** [comment]
Thanks for quick answer.

I did make a small piece of code calling module appium.
Nuitka produced an 'exe' which work without issue. It is weird.

So I execute again my main program and I see the line of code which create the issue:

```
Traceback (most recent call last):
  File "E:\DropboxBACKUP14112018\Cff\Python\PhoneBot_with_lock\phonebot.dist\phonebot.py", line 7, in <module>
  File "mymodules.py", line 18, in init mymodules
ModuleNotFoundError: No module named 'appium'
```

In my short test code, I have exactly same line of code than this line 18:

`from appium import webdriver`

So the question is why this line of code is going fine in the short code, but it creates a "ModuleNotFoundError: No module named 'appium'" error in my main code?

I attached here the full logs of my manip.

[https://github.com/gauthierbuttez/public/blob/master/nuitka_log2.txt](https://github.com/gauthierbuttez/public/blob/master/nuitka_log2.txt)

So I tried something: I added the line "import appium" above the line "`from appium import webdriver` in order to import the entire module "appium".

I didn't see this error module anymore. But my "exe" is opening several thread, one by one without ending. Something very weird.

When I execute my python script from command line, it works perfectly.
The exe is doing very weird things.

So as I saw several new thread opening one by one in my table of threads (CTRL+AT+DEL), I thought "multiprocessing".

I check for any nuitka plugin, and it exist a plugin for multithreading.
This was my final command line:

`python -m nuitka --standalone --plugin-enable=numpy=xarray --show-progress --show-scons --follow-imports --recurse-all --windows-icon=E:\\favicon.ico myprogram.py`

Thank you so much for your support.

If one day a user doesn't have any outputt, anything at all but he can see several threads of his program.exe created in the table of threads, it is an issue of multiprocessing. He need to activate plugin.

I have now some issues like this one
"TypeError: '>' not supported between instances of 'NoneType' and 'NoneType'".
I will try to investigate myself and certainly come back in an new issue. I am coming closer to the end. :-)
